### ISSUE TITLE
Hero rankings have some randomness to them.

### ISSUE THREAD

**Issue by Darkitz** [odota/core#1635]
Upon refreshing my hero rankings page i keep getting randomized results. (atleast for my top heroes).

The number are so far apart that this is an unusual fluctuation (top 100 to top 4000)

reference: https://www.opendota.com/players/60372773/rankings

**Comment by howardchung** [comment]
For performance reasons the rankings are done against a random sample on each request, so depending on the sample the results may vary.  I can look into increasing the sample size to reduce the variance, but we'll need to balance that against performance.

**Comment by howardchung** [comment]
I doubled the sample size, is it more consistent now?

**Comment by Darkitz** [comment]
Yep, a bit better. But i honestly dont think the tradeoff is worth it.  

Probably have to overhaul the system in long-term.
I'll look into it (with my limited knowledge) of what i can do, over the next few weeks.
